### DISCUSSION TITLE
Expose only _some_ class member functions?

### DISCUSSION THREAD

**Discussion by jwuttke** [RosettaCommons/binder#206] [General]
Exists there already a mechanism to expose only some, not all member functions of a class to Python?

If not, do you have suggestions how such a mechanism could work? How to flag functions for exclusion? With preprocesor conditionals as in Swig?

**Comment by lyskov** [comment]
Yes, - one can disable bindings for particular member function using `-function` config directive which is documented here: https://cppbinder.readthedocs.io/en/latest/config.html#config-file-options Please let me know if you need more info on this.
